x-pack/plugins/cross_cluster_replication/index.js

```diff
@@ -38,7 +38,15 @@ export function crossClusterReplication(kibana) {
         enabled: Joi.boolean().default(true),
       }).default();
     },
-
+    isEnabled(config) {
+      return (
+        config.get('xpack.ccr.enabled') &&
+        config.has('xpack.index_management.enabled') &&
+        config.get('xpack.index_management.enabled') &&
+        config.has('xpack.remote_clusters.enabled') &&
+        config.get('xpack.remote_clusters.enabled')
+      );
+    },
```

Comment:
CCR's UI surfaces cross-links with Remote Clusters. Should we also set `ccrUiEnabled` to `false` on line 25 if Remote Clusters UI is disabled?

```js
        return {
          ccrUiEnabled: (
            config.get('xpack.ccr.ui.enabled')
            && config.get('xpack.remote_clusters.ui.enabled')
          )
        };
```